src/shared/components/common/markdown-textarea.tsx

```diff
@@ -431,13 +431,34 @@ export class MarkdownTextArea extends Component<
   }
 
   // Keybind handler
+  // Keybinds inspired by github comment area
   handleKeyBinds(i: MarkdownTextArea, event: KeyboardEvent) {
     if (event.ctrlKey) {
       switch (event.key) {
-        case "k": {
-          // Currently only one case but will support further keybinds
+        case "k": // Insert link
           i.handleInsertLink(i, event);
-        }
+          break;
+        case "b": // Insert bold
+          i.handleInsertBold(i, event);
+          break;
+        case "i": // Insert italic
+          i.handleInsertItalic(i, event);
+          break;
+        case "e": // Insert code
+          i.handleInsertCode(i, event);
+          break;
+        case "8": // Insert unordered list
+          i.handleInsertList(i, event);
+          break;
+        case "s": // Insert spoiler
+          i.handleInsertSpoiler(i, event);
+          break;
+        case "p": // Move to preview view
```

Comment:
Nitpick: The comments are unnecessary because the function names already make it clear what is going on.

Reply:
Done. I think I was being a bit overkill ha ha